### ISSUE TITLE
Mac with the new font rendering...

### ISSUE THREAD

**Issue by SeamusFD** [Facepunch/garrysmod-issues#1851]
I have got to admit the new fonts in the recent patch look nice, however that is only when they work.
My garry's mod console cannot find the Lucida font and reverts to the Tahoma which quite frankly looks horrible. Note that I am on Mac OS X Yosemite.
This is the error log from my console when it tries to load the font for the console.
http://pastebin.com/ptamfx2s.

I would love to fix this, and I am also concerned that this is connected to a similar game-breaking crash such as the one described here: #1836.

Any insight in this matter would be appreciated.

**Comment by Bo98** [comment]
Open the Font Book application and check if you have the font.

**Comment by Bo98** [comment]
I have a potential fix for this if you find Lucida Console isn't installed by default. However #1852 prevents it from working without crashing on startup.

**Comment by samgillie** [comment]
I just checked.   You are right !   Lucida Console is not in the Font book by default !

**Comment by Bo98** [comment]
Right, I will send in a pull request to fix it when #1852 is fixed.

**Comment by samgillie** [comment]
http://www.ffonts.net/Lucida-Console.font.download 
double click the font file and click Install font.   :)

**Comment by SeamusFD** [comment]
Ok the problem is resolved... However it does not fix my crashing
